charts/backstage/values.yaml

```diff
@@ -308,3 +308,18 @@ serviceAccount:
 
   # -- Auto-mount the service account token in the pod
   automountServiceAccountToken: true
+
+# ServiceMonitor configuration
+serviceMonitor:
+  # -- If enabled, ServiceMonitor resources for Prometheus Operator are created
```

Comment:
Please advise users that Prometheus Operator must be already present on the cluster prior to installing this chart with this enabled (aka. enabling this makes the `ServiceMonitor` CR a prerequisite).